auto_submit/bin/server.dart

```diff
@@ -9,25 +9,27 @@ import 'package:auto_submit/helpers.dart';
 import 'package:shelf/shelf.dart';
 import 'package:shelf_router/shelf_router.dart';
 
-class AutosubmitServer {
-  AutosubmitServer();
+Future<Response> webhookHandler(Request request) async {
```

Comment:
This should be moved into it's own file under lib/ like `lib/requests/github_webhook.dart`